pages/user.js

```diff
@@ -0,0 +1,46 @@
+import Cookies from "cookies";
+import Iron from "@hapi/iron";
+import { ENC_KEY } from "./api/users";
+import Link from "next/link";
+export async function getServerSideProps(context) {
+  const cookies = new Cookies(context.req, context.res);
+  const sessionstr = cookies.get("session");
+
+  if (sessionstr) {
+    try {
+      const session = await Iron.unseal(sessionstr, ENC_KEY, Iron.defaults);
+      if (session.loggedIn) {
+        return {
+          props: {
+            username: session.username,
+          },
+        };
+      }
+    } catch (err) {}
+  }
+  return {
+    notFound: true,
+  };
+}
+
+const UserPage = ({ username }) => {
+  return (
+    <div className="userDiv">
+      <h1 style={{ color: "white" }}>You are now in the user page </h1>
+      <h2 className="userWelcom">
+        Welcome:
+        <a className="welcomeA">{username}</a>
```

Comment:
Is this element going to be linked somewhere in the future? Even if so, until then we might not use a hyperlink element. And might consider a more consistent naming of this className _(userWelcomeLink)?_

Reply:
good idea, I will update it.